_layouts/site.html

```diff
@@ -31,7 +31,7 @@
     hljs.registerLanguage("redshift", sql.rawDefinition);
     hljs.highlightAll();
   </script>
-
+  <link rel="icon" type="image/x-icon" href="/advanced-mis/images/favicon.ico">
```

Comment:
I've added this here as otherwise the page looks for the favicon in `/favicon.ico` but the `/advanced-mis` bit is needed (I'm assuming because of the [base url set in the config](https://github.com/talis/advanced-mis/blob/master/_config.yaml#L3)). If there's a better way of making this work please let me know.

Reply:
the new favicon looks good to me!